### PR TITLE
CCLOG-1274: Avoid logging config, data by default

### PR DESCRIPTION
**Pull request by ragepati**
## Problem
The configuration or data should not be logged by default

## Solution
Reduce the log levels so that this data is not logged by default.

##### Does this solution apply anywhere else?
- [ ] yes
- [x] no

### CODE REVIEW

File: src/main/java/com/splunk/kafka/connect/SplunkSinkConnector.java [hunk 1]
```diff
@@ -133,12 +133,12 @@ private void validateAccess(SplunkSinkConnectorConfig connectorConfig, Map<Strin
                 try (CloseableHttpResponse response = httpClient.execute(request)) {
                     status = response.getStatusLine().getStatusCode();
                     if (status == 400) {
-                        log.info("Validation succeeded for indexer {}", uri);
+                        log.trace("Validation succeeded for indexer {}", uri);
```
Comment (mukkachaitanya) [inline src/main/java/com/splunk/kafka/connect/SplunkSinkConnector.java:136]: Losing these logs can make it difficult to debug? Can we only remove the indexer URI from the log instead? or maybe print the indexer index in the config. Like `Validation succeeded for indexer 3`

Comment (mukkachaitanya) [reply src/main/java/com/splunk/kafka/connect/SplunkSinkConnector.java:136]: Also, in the same spirit Line [151](https://github.com/confluentinc/kafka-connect-splunk/pull/7/files#diff-4b2cb575dc12d52bd44c1e35194c9e7b211a8879025030cf94d9768027754a4cR151) also seems to be logging the URIs?

Comment (mukkachaitanya) [reply src/main/java/com/splunk/kafka/connect/SplunkSinkConnector.java:136]: Further, the Validation string would also contain the URI which might be logged elsewhere in the pipeline, is that of concern too?

Comment (ragepati) [reply src/main/java/com/splunk/kafka/connect/SplunkSinkConnector.java:136]: That's a good idea. I'll add the indexing back in with a separate change. Thanks!

**Review by tunnelshade — APPROVED** [review]

**Review by mukkachaitanya — APPROVED** [review]

**Pull request merged**